src/components/side_panel/chart/building_blocks/title/title.xml

```diff
@@ -1,11 +1,11 @@
 <templates>
   <t t-name="o-spreadsheet.ChartTitle">
     <t t-set="placeholder">Add a title</t>
-    <Section class="'o-chart-title'">
-      <t t-set-slot="title">
-        <t t-if="props.name" t-esc="props.name"/>
-        <t t-else="">Title</t>
-      </t>
+    <t t-set="title">
+      <t t-if="props.name" t-esc="props.name"/>
+      <t t-else="">Title</t>
+    </t>
+    <Section class="'o-chart-title'" title="title.toString()">
```

Comment:
what's the toString for ?

Reply:
It's because of the `t-set`. Owl compiles it to a LazyValue which not an `instanceof String` and props validation fails :/
It feels like it should/could be improved in owl code but bon :shrug: 

https://github.com/odoo/owl/blob/3e11fe6b12a74a6081f79059f6822ebe9c23a45c/src/compiler/code_generator.ts#L1096